tasks/libs/junit_upload.py

```diff
@@ -246,3 +250,32 @@ def repack_macos_junit_tar(infile, outfile):
         # pack all files to a new tarball
         for f in os.listdir(tempd):
             outfp.add(os.path.join(tempd, f), arcname=f)
+
+
+def retrieve_jira_card(xmlfile_path):
+    """
+    Search in jira if a card already exist for the given testsuite
+    """
+    from .pipeline_notifications import send_slack_message
+    from jira import JIRA
```

Comment:
Pending addition of this module to the datadog-ci-uploader image, see https://github.com/DataDog/datadog-agent-buildimages/pull/438

Reply:
Included in latest image